CourseStream/CourseStream.Data/Migrations/20231122174018_AddEnrollmentTimestamp.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CourseStream.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddEnrollmentTimestamp : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTimeOffset>(
+                name: "EnrollmentTimestamp",
+                table: "EnrolledStudents",
+                type: "timestamp with time zone",
+                nullable: false,
+                defaultValue: new DateTimeOffset(new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified), new TimeSpan(0, 0, 0, 0, 0)));
```

Comment:
DateTimeOffset.UtcNow will be better, but not sure it will work correctly. Need to validate this theory